scripts/github-repos/README.md

```diff
@@ -1,53 +1,138 @@
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -o, --orgname=ORGNAME            The name of the org
+
+'indiv_repos'
+Create CHIPS repo for each stedent in STUDENTTEAM. Repos' names are form like "PREFIX-[username]-FILENAME"
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -s, --studentteam=STUDENTTEAM    The team name of all the students team
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -t, --template=TEMPLATE          The repo name within the org to use as template
     -g, --gsiteam=GSITEAM            The team name of all the staff team
+    -f, --filename=FILENAME          The base filename for repos
+
+'team_repos'
+Create 10.5 repos for each child team. Repos' names are form like "PREFIX-FILENAME-[Team number]"
+Make sure child teams are formed before running this command.
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -t, --template=TEMPLATE          The repo name within the org to use as template
+    -f, --filename=FILENAME          The base filename for repos.
+    -s, --studentteam=STUDENTTEAM    The team name of all the students team
+    -g, --gsiteam=GSITEAM            The team name of all the staff team
+
+'remove_indiv_repos'
+Delete all repos whose names are formed like "PREFIX-[username]-FILENAME".
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -f, --filename=FILENAME          The base filename for repos.
+
+'remove_team_repos'
+Delete all repos whose names are formed like "PREFIX-FILENAME-[Team number]".
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -f, --filename=FILENAME          The base filename for repos.
+
+'remove_teams'
+Remove all students and child teams in STUDENTTEAM from the org.
+Remove STUDENTTEAM as well.
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -s, --studentteam=STUDENTTEAM    The team name of all the students team
+
+'remove_team_repo_access'
+Remove students access to CHIP 10.5 repos that are formed like "PREFIX-FILENAME-[Team number]".
+
+Required options:
+    -o, --orgname=ORGNAME            The name of the org
+    -p, --prefix=PREFIX              Semester prefix, eg fa23.
+    -f, --filename=FILENAME          The base filename for repos.
 ```
 
 This script creates a team that include all stundents in the same semester, eg fa23. 
 Then creates different child teams for them for the chip10.5. At a minimum, 
-you need a CSV file listing all enrolled students with columns 'Email' and 'Team'.  
+you need a CSV file listing all enrolled students with columns 'Username' and 'Team'.  

```

Comment:
does the Username column need to have the student's personal GitHub username, or something else?

Reply:
The Username column need to have the student's personal GitHub username.  Since student will have emails other than berkerley email associated with GitHub. I think using GItHub username is better to invite students to org.